experimental/examples/tree-sample/inventory-app/src/domains/model/schema.ts

```diff
@@ -0,0 +1,19 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { FieldKinds, fieldSchema, rootFieldKey, SchemaData } from "@fluid-internal/tree";
+
+import { float64 } from "./primitives";
+import { inventory } from "./inventory";
+
+const rootField = fieldSchema(FieldKinds.value, [inventory.name]);
+
+export const schema: SchemaData = {
```

Comment:
This seems important, probably add comments for viewers of the example to understand what's going on.

Reply:
We're actively working to streamline schema declaration, but I can add a quick note for interested parties following the evolution.